docs/source/index.rst

```diff
@@ -6,6 +6,13 @@
 Welcome to MatFlow's documentation!
 ===================================
 
+MatFlow is an open source Python framework for managing computational materials science workflows.
```

Comment:
This bit needs checking by @aplowman , I've described it as best I know but I probably have some misunderstandings, and you might want different links?